WBooks/src/app/screens/BookDetails/screens/BookDetails/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, ScrollView, SafeAreaView, TouchableOpacity } from 'react-native';
+
+import BookImage from '../../../../components/BookImage';
+
+import styles from './styles';
+
+function BookDetails({ route }) {
+  const { book } = route.params;
+  const { title, author, year, genre, imageUrl } = book;
+  return (
+    <SafeAreaView>
+      <ScrollView style={styles.bookDetails}>
+        <View style={styles.container}>
+          <View style={styles.bookInfoContainer}>
+            <BookImage style={styles.bookImage} url={imageUrl} />
+            <View style={styles.bookInfo}>
+              <Text style={styles.title}>{title.trim()}</Text>
```

Comment:
Should I configure this title when I get the bookList? or is it ok to trim the string each time I use it?

Reply:
In this case its ok because this component should render just one or two times because is a Pure component and his props is never changing.